hadoop-hdds/docs/content/security/SecurityAcls.md

```diff
@@ -172,3 +172,16 @@ $ ozone sh bucket removeacl -a user:testuser:r[DEFAULT] /vol1/bucket2
 ACL user:testuser:r[DEFAULT] removed successfully.
 ```
 
+## Differences Between Ozone ACL and S3 ACL
+
+Ozone ACLs and S3 ACLs differ primarily in their scope and support.
+
+- **S3 ACLs**: Currently, only S3 Bucket ACL is supported in Ozone. S3 Object ACL is not yet implemented. Any `PutObjectAcl` request will result in a `501: Not Implemented` response code.
+- **Ozone ACLs**: Ozone ACLs provide a more comprehensive and flexible access control mechanism. They are designed to work seamlessly with Ozone's native architecture and support various rights and scopes as mentioned above.
+
+## Ozone File System ACL API
+
+- ACL-related APIs in Ozone file system implementation (`ofs` and `o3fs`), such as `getAclStatus`, `setAcl`, `modifyAclEntries`, `removeAclEntries`, `removeDefaultAcl`, and `removeAcl` are not supported. These operations will throw an UnsupportedOperationException.
```

Comment:
IIRC, currently it will return some instantly created ACL, instead of UnsupportedOperationException.

Reply:
which one? https://github.com/apache/ozone/pull/8458/files
i added tests to confirm all throw UnsupportedOperationException.
the test doesn't include getAclStatus() but from a quick look at BasicRootedOzoneFileSystem, it doesn't look like it's implemented, and the default FileSystem.getAclStatus() throws UnsupportedOperationException